packages/vuetify/src/components/VBanner/VBannerActions.tsx

```diff
@@ -0,0 +1,32 @@
+// Composables
+import { provideDefaults } from '@/composables/defaults'
+
+// Utility
+import { defineComponent, useRender } from '@/util'
+
+export const VBannerActions = defineComponent({
+  name: 'VBannerActions',
```

Comment:
This is a pretty useless component anyway, you could replace it with VDefaultsProvider and a div